tango_simlib/sim_xmi_parser.py

```diff
@@ -448,6 +448,10 @@ def _get_arg_type(self, description_data):
             if arg_type in ['FloatArray', 'DoubleArray', 'StringArray']:
                 arg_type = getattr(PyTango, 'DevVar' + arg_type)
             else:
+                # Substituting the 'Int' type with 'Long'. 'DevInt' is not a supported
+                # data type in TANGO.
+                if arg_type == 'Int':
+                    arg_type = 'Long'
```

Comment:
Here's what POGO is doing with the data types on the GUI:  https://github.com/tango-controls/pogo/blob/8bf0a50/org.tango.pogo.pogo_gui/src/org/tango/pogo/pogo_gui/tools/OAWutils.java#L607-L654, if you want to compare.